app/cafe.py

```diff
@@ -0,0 +1,23 @@
+import datetime
+from app.errors import (
+    VaccineError,
+    NotVaccinatedError,
+    OutdatedVaccineError,
+    NotWearingMaskError
+)
+
+
+class Cafe:
+
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        current_date = datetime.date.today()
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError("NotVaccinatedError")

```

Comment:
The error message 'NotVaccinatedError' is not descriptive. According to the checklist, you should provide a more descriptive error message when raising exceptions. Consider including more context in the message, such as 'Visitor is not vaccinated.'